pom.xml

```diff
@@ -20,6 +20,7 @@
 		<module>spring-grpc-test</module>
 		<module>spring-grpc-spring-boot-autoconfigure</module>
 		<module>spring-grpc-spring-boot-starter</module>
+        <module>spring-grpc-client-spring-boot-starter</module>
```

Comment:
There is something off w/ my xml format settings. I will get these fixed here in this PR and push another commit.